luminoth/utils/predicting.py

```diff
@@ -109,8 +114,12 @@ def predict_image(self, image):
         if self.class_labels is not None:
             labels = [self.class_labels[label] for label in labels]
 
-        # Scale objects to original image dimensions.
-        objects /= scale_factor
+        # Scale objects to original image dimensions
+        if isinstance(scale_factor, list):
+            objects /= [scale_factor[1], scale_factor[0],
```

Comment:
Add comment on what scale factor is when it is not a single value. (Also, maybe use tuple instead of list?)